packages/sdk-auth/src/auth.js

```diff
@@ -81,19 +81,31 @@ export default class SdkAuth {
     return Buffer.from(`${clientId}:${clientSecret}`).toString('base64')
   }
 
+  static _getScopes(scopes: ?Array<string>, projectKey: ?string) {
+    return scopes
+      ? scopes.join(' ')
+      : [constants.MANAGE_PROJECT, projectKey].filter(Boolean).join(':') // generate a default scope manage_project:projectKey
```

Comment:
This is because if you don't provide a `projectKey` it will build an incorrect scope name.
In some cases you can provide scopes also like this:
```
scope=view_users_and_organizations view_projects
```
And API will create a token with these limitations for all your projects.